_includes/custom/logging-flags.md

```diff
@@ -0,0 +1,9 @@
+Flag | Description
+-----|------------
+`--log-dir` | Enable logging to files and write logs to the specified directory.<br/><br/>Including the `--log-dir` flag without specifying a directory (e.g., `cockroach start --log-dir`) or setting it to a blank directory (`--log-dir=""`) disables logging to files.
```

Comment:
"Including the --log-dir flag without specifying a directory [...] disables logging to files" -> this is incorrect.

If you include "--log-dir" then whatever comes afterwards will become the logging directory. For example if you write `--log-dir --insecure` then your logging directory becomes `/path/to/your/cwd/--insecure`. Hardly what you want. :)
I recommend stating that `--log-dir` always takes a string argument, and that the empty string disables logging to files.